src/Lecture3.hs

```diff
@@ -126,9 +145,14 @@ data Reward = Reward
     } deriving (Show, Eq)
 
 instance Semigroup Reward where
-
+  (<>) x y = Reward { rewardGold = rewardGold x <> rewardGold y
+                    , rewardSpecial = rewardSpecial x || rewardSpecial y
+                    }
 
 instance Monoid Reward where
+  mempty = Reward { rewardGold = Gold 0
```

Comment:
You can use `mempty` for `Gold` here because it already has a `Monoid` instance 🏅 
```suggestion
  mempty = Reward { rewardGold = mempty
```

Reply:
I see. 👍🏼